uwblueprint/ccbc
# Add filters to homepage.

## Notion ticket link

[Home Page Filtering](https://www.notion.so/uwblueprintexecs/Home-Page-Filtering-5f21c02b0799475ab739e74f9c9a47af)



## Implementation description
* Added existing filtering code to the homepage



## Steps to test
1. Attempt to select some filters and see if they are applied

Note: I had trouble testing this because the existing filtering options are empty (in `/magazine/search_results`). We should fix this as well.

## Checklist
